cmd/runtimecfg/node-ip.go

```diff
@@ -269,3 +270,8 @@ func parseIPs(args []string) ([]net.IP, error) {
 	}
 	return ips, nil
 }
+
+// currently we allow setting remote worker only in case of baremetal platform without external lb
+func setRemoteWorker(vips []net.IP, matchesVips bool) bool {
```

Comment:
Can we rename from `set` to `is`? This function as I see it is only answering a question and is not setting any state anywhere